.github/workflows/run-tests-on-pull-request.yml

```diff
@@ -37,5 +37,15 @@ jobs:
         env:
           ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
         run: docker build --build-arg ECR_REGISTRY=$ECR_REGISTRY -f Dockerfile.tests -t pos-network-node:test .
+      - name: Slack Notification
+        if: ${{ always() }}
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_CHANNEL: e2e-test-results
```

Comment:
Integration tests are not e2e tests. We should think about the channels hierarchy for notifications. 1 channel can be used but in this case we should use mentioning. Or we should think about an independent channel for each component/product.

Reply:
@AndreiNavoichyk we have separate bots
`SLACK_USERNAME: ddc-smart-contract-integration-tests-bot`

for this name, we can identify the test, 
you can see all test results in one channel in my opinion, 
it is not a good idea to add a few test results channels, focus on one, and when we have so much tests and it will be a problem we will add separate channels.